configure.sh

```diff
@@ -1,11 +1,10 @@
 #!/bin/bash
 
 # Default values for all tunables.
-CIRCT_PATH=
+CIRCT_PATH=${PWD}/build-circt/circt
```

Comment:
If I remember correctly this should be in quotes in case the person running the script has spaces in one of the parent folders